web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/arbeidInntektsmelding/ArbeidOgInntektsmeldingMapper.java

```diff
@@ -83,8 +82,7 @@ public static InntektsmeldingDto mapInntektsmelding(Inntektsmelding im,
                 im.getNaturalYtelser(),
                 im.getEndringerRefusjon(),
                 im.getInntektsmeldingInnsendingsårsak(),
-            behandlingsIder,
-            aktiveBehandlingsIder
+            aktiveBehandlingsIder != null ? aktiveBehandlingsIder : List.of() // TODO: sjekk null her eller før??
```

Comment:
Usikker hvor dette bør gjøres. Tenker det gir mest mening å gjøre i mappinga for å sikre tom array og ikke null?